fortio_main.go

```diff
@@ -226,6 +231,92 @@ func main() {
 	}
 }
 
+func generateJSONFile(res periodic.HasRunnerResult, out io.Writer) {
```

Comment:
while that code was already somewhat present and already somewhat untested (or at least not tested through unit tests; do check WebTest.sh for e2e tests...) it'd be nice to ideally move that function in one of the util pkg and test it there